ykman/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Contributor: Daniel Everett <deverett@gmail.com>
+# Maintainer: Daniel Everett <deverett@gmail.com>
+pkgname=yubikey-manager
+pkgver=0.7.1
+pkgrel=0
+pkgdesc="Python library and command line tool for configuring any YubiKey over all USB interfaces"
+url="https://github.com/dae3/alpine-yubikey-manager"
+arch="noarch"
+license="BSD-2-Clause"
+depends="python3 py-setuptools libusb yubikey-personalization-dev libyubikey pcsc-lite pcsc-lite-libs"
```

Comment:
depends= only used when some package needs a runtime dependency. yubikey-personalization-dev seems not a runtime dependency since it is a development package. You have to strictly apart runtime dependencies (depends=) and build time dependencies (makedepends=).